src/components/modalWindowContent/modalWindowContent.js

```diff
@@ -0,0 +1,281 @@
+    correctAnswer: 'Beru Whitesun lars',
+    human: {
+      answer: 'Beru Whitesun lars',
+      isCorrect: true
+    },
+    autoPlayer: {
+      answer: 'Darth Vader',
+      isCorrect: false
+    }
+  },
+  {
+    id: 3,
+    img: '../../../static/assets/img/modes/people/19.jpg',
+    correctAnswer: 'Jek Tono Porkins',
+    human: {
+      answer: 'Chewbacca',
+      isCorrect: false
+    },
+    autoPlayer: {
+      answer: 'Jek Tono Porkins',
+      isCorrect: true
+    }
+  },
+  {
+    id: 4,
+    img: '../../../static/assets/img/modes/people/26.jpg',
+    correctAnswer: 'Lobot',
+    human: {
+      answer: 'Lobot',
+      isCorrect: true
+    },
+    autoPlayer: {
+      answer: 'Darth Vader',
+      isCorrect: false
+    }
+  }
+]; */
+
+const hideModal = () => {
+  const modal = document.querySelector('.content');
+  modal.classList.add('hidden');
+};
+
+const countCorrectAnswers = (answersList, player) => {
+  const answers = [];
+  answersList.forEach((answer) => {
+    if (answer[player].isCorrect) {
+      answers.push(answer[player].isCorrect);
+    } else return;
+  });
+  return answers;
+};
+
+const getYodaQuote = (humanPlayer, autoPlayer) => {
+  if (humanPlayer > autoPlayer) {
+    return 'The force is strong in you young Padawan!';
+  } else return 'Much to learn you still have...my young Padawan.';
+};
+
+const getStringForEndGame = (
+  quote,
+  questionsNumber,
+  humanAnswers,
+  autoPlayerAnswers
+) => {
+  return `${quote} During 1 minute you have answered ${humanAnswers} / ${questionsNumber} questions. And Google guessed ${autoPlayerAnswers} / ${questionsNumber}.`;
+};
+
+const getTextForEndGame = (answersList) => {
+  const questionsNumber = answersList.length;
+
+  const human = countCorrectAnswers(answersList, 'human').length;
+  const autoPlayer = countCorrectAnswers(answersList, 'autoPlayer').length;
+
+  const text = getStringForEndGame(
+    getYodaQuote(human, autoPlayer),
+    questionsNumber,
+    human,
+    autoPlayer
+  );
+  return text;
+};
+
+const createTitle = () => {
+  const title = document.createElement('h2');
+  title.classList.add('content_title');
+  title.innerText = 'GAME OVER';
+  return title;
+};
+
+const createSummary = (answersList) => {
+  const summary = document.createElement('p');
+  summary.classList.add('content_summary');
+  summary.innerText = getTextForEndGame(answersList);
+  return summary;
+};
+
+const createHeroImg = () => {
+  const img = new Image();
+  img.classList.add('middle-wrapper_img');
+  img.src = '../../static/assets/ui/MasterYodaLeft.png';
+  img.alt = 'MasterYoda';
+  return img;
+};
+
+const createAnswersHeading = () => {
+  const answersHeading = document.createElement('h3');
+  answersHeading.classList.add('right-wrapper_title');
+  answersHeading.innerText = 'Detailed answers:';
+  return answersHeading;
+};
+
+const createTable = (answersList) => {
+  const table = document.createElement('table');
+  table.classList.add('table');
```

Comment:
you used BEM before and it looked great, why didn't you use this methodology here?

Reply:
what I expect here is more proper name, table doesn't say anything
better will be to set class name similar to component name e.g.: `resultsTable` `resultsTable__header` `resultsTable__body` etc.

Reply:
I don't know :P